extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -210,6 +212,19 @@
   display: block !important;
 }
 
+.i-amphtml-story-mobile .i-amphtml-paused-display {
+  display: none !important;
+}
+
+.i-amphtml-story-system-layer:not([i-amphtml-current-page-has-playable]) .i-amphtml-paused-display * {
+  opacity: 0.3;
```

Comment:
what happens when/if we add a new icon next to the pause/play and this is not displayed? would we have an empty space in between?

Reply:
When it's not displayed (ie. in mobile), the icon does not take up space. Currently the play/pause is located between share and volume, and when it's not active, there's no extra space between the other two icons. The opacity changes mean that the icon will be disabled when the current page doesn't have a playable element but the story does, so as to avoid UI jumps between pages. If the whole story doesn't have anything that can be played/paused, then the icon doesn't even show (and there's no space between the audio and share icons). Let me know if you have any more questions.

Reply:
You can check the demo on the description, just added it for reference.

Reply:
Nit: please add an `!important` statement so publishers can't override this. It's in a shadow DOM, but might still be exposed in some older browsers that don't support shadow DOM.